bindings/python/flashlight/lib/text/decoder/__init__.py

```diff
@@ -29,8 +29,3 @@
     TrieNode,
     ZeroLM,
 )
-
-try:
-    from .flashlight_lib_text_decoder import KenLM
```

Comment:
If you want to keep the previous name convention for KenLM, while performing lazy-import, you can define `__getattr__` on module level and let it handle KenLM differently.

https://twitter.com/raymondh/status/1094686528440168453?lang=en

We used this for functionalities which require optional dependencies. (though we migrated to different mechanism as these functions are now used in other parts of libraries, so import happens anyways.)

https://github.com/pytorch/audio/blob/2ead941e398712f0d90f41ba7b88325e04725ebe/torchaudio/io/__init__.py

Reply:
Actually, from the viewpoint of smooth transition, keeping the same naming convention is ideal.

https://github.com/pytorch/audio/pull/3055

Reply:
Talked offline -- swapped to `__getattr__` with a `PendingDeprecationWarning` to preserve BC.